test/cloudtest/pkg/commands/main.go

```diff
@@ -1303,8 +1362,9 @@ func (ctx *executionContext) generateClusterFailedReportEntry(inst *clusterInsta
 
 func (ctx *executionContext) generateTestCaseReport(test *testTask, totalTests int, totalTime time.Duration, failures int, suite *reporting.Suite) (int, time.Duration, int) {
 	testCase := &reporting.TestCase{
-		Name: test.test.Key,
-		Time: test.test.Duration.String(),
+		Name:    test.test.Key,
+		Time:    test.test.Duration.String(),
```

Comment:
It seems we need to convert time to seconds, CircleCI doesn't understand it properly.